Yoast/wordpress-seo
# Removed Remind me in 7 days button in indexing notification

## Context


* Removed "Remind me in 7 days" option in indexation notification.

## Summary


This PR can be summarized in the following changelog entry:

* Removes the "Remind me in 7 days" option in the indexation notification, which was shown for sites with 2500+ unindexed indexables.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure you have more than 2500 indexable entities.
  * You can do this using the following wp-cli command:
    ```
    wp posts generate --count=2600
    ```
* Reset the indexables so the notification is shown.
* If the notification does not show up due to the cron job, add
  ```php
  \add_action( 'admin_init', function(){ do_action( 'wpseo-reindex' ); }, 100 );
  ```
  to your `functions.php` file.
* Go to the Yoast SEO dashboard in the WordPress admin (_SEO_ > _General_)
* There should **not** be a link to hide the notification for a week in the notification to reindex your site.
<img width="607" alt="Screenshot_2020-10-19_at_09_36_19" src="https://user-images.githubusercontent.com/10195175/96415455-ec57d580-11ee-11eb-8d34-870e436ada83.png">


## UI changes

## Documentation

## Quality assurance


Fixes #
